src/index.js

```diff
@@ -1,52 +1,17 @@
 import './style.css';
+import GameScore from './modules/gameScore.js';
 
-const list = [
-  {
-    id: '1',
-    name: 'name',
-    score: '100',
-  },
-  {
-    id: '2',
-    name: 'name',
-    score: '20',
-  },
-  {
-    id: '3',
-    name: 'name',
-    score: '50',
-  },
-  {
-    id: '4',
-    name: 'name',
-    score: '78',
-  },
-  {
-    id: '5',
-    name: 'name',
-    score: '125',
-  },
-  {
-    id: '6',
-    name: 'name',
-    score: '77',
-  },
-  {
-    id: '7',
-    name: 'name',
-    score: '42',
-  },
-];
+const currentScore = new GameScore();
+const addScore = document.querySelector('.add-form');
 
-const listItem = document.getElementById('list');
+addScore.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const user = addScore.name.value;
+  const score = addScore.score.value;
+  currentScore.addNewScore({ user, score });
+  addScore.reset();
+});
 
-list.map((item) => {
-  if (item.id % 2 === 0) {
-    listItem.innerHTML += `
-    <li class= 'nbgGrey'>${item.name}:<span>${item.score}</span></li>`;
-  } else {
-    listItem.innerHTML += `
-<li>${item.name}:<span>${item.score}</span></li>`;
-  }
-  return ('');
-});
\ No newline at end of file
+const refreshBtn = document.getElementById('refresh');
```

Comment:
- [ ] [OPTIONAL] I think the scores should load-up up on entering the app, not until it is refreshed. the refresh I think is for when the user is offline, then comes back online to get the latest scores.

![mobile](https://user-images.githubusercontent.com/41163571/233363798-bb115509-256b-4def-a8fa-0e67a2923764.gif)

Reply:
I didn't know that was the case. I will look into it now.
Thank you.